src/covid_model_seiir_pipeline/pipeline/diagnostics/model/plot_version.py

```diff
@@ -106,9 +106,14 @@ def clean_data(self, data: pd.DataFrame, location_id: Optional[int]):
 
 
 def make_plot_versions(comparators: List[GridPlotsComparatorSpecification], color_map) -> List[PlotVersion]:
+    primary_versions = []
     plot_versions = []
     for comparator in comparators:
         for scenario, label in comparator.scenarios.items():
-            plot_versions.append((Path(comparator.version), scenario, label))
-    plot_versions = [PlotVersion(*pv, color_map(i)) for i, pv in enumerate(plot_versions)]
-    return plot_versions
+            if scenario.endswith('*'):
```

Comment:
ok, I get the asterisks now. I'm cringing a little, but not too much...

Reply:
Sorry not sorry.